WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -50,17 +55,23 @@ class SiteStatsInsightsTableViewController: UITableViewController, StoryboardLoa
     static var defaultStoryboardName: String = "SiteStatsDashboard"
 
     // MARK: - Properties
+
     private var insightsChangeReceipt: Receipt?
 
-    // TODO: update this array when Manage Insights is implemented.
     // Types of Insights to display. The array order dictates the display order.
     private var insightsToShow = [InsightType]()
     private let userDefaultsInsightTypesKey = "StatsInsightTypes"
 
-    // Store customize separately as it is not per site.
-    // (insightsToShow is not yet per site, but it will be.)
+    // Store 'customize' separately as it is not per site.
     private let userDefaultsHideCustomizeKey = "StatsInsightsHideCustomizeCard"
     private var hideCustomizeCard = false
+
+    // Store Insights settings for all sites.
+    // Used when writing to/reading from User Defaults.
+    // A single site's dictionary contains the InsightType values for that site.
+    private var allSitesInsights = [siteInsights]()
+    private typealias siteInsights = [String: [Int]]
```

Comment:
Nitpick, but I think the standard is to capitalise typealias type names (`SiteInsights`)

Reply:
Woops. Thanks! Tis been changed.